master/buildbot/db/buildrequests.py

```diff
@@ -148,9 +152,12 @@ def getTotalBuildsInTheLastDay(self):
         def thd(conn):
             reqs_tbl = self.db.model.buildrequests
             yesterday = datetime.now().date() - timedelta(1)
-            lastday = datetime2epoch(datetime(yesterday.year, yesterday.month, yesterday.day))
+            beforeyesterday = datetime.now().date() - timedelta(2)
+            yesterday_epoch = datetime2epoch(datetime(yesterday.year, yesterday.month, yesterday.day))
+            beforeyesterday_epoch = datetime2epoch(datetime(beforeyesterday.year, beforeyesterday.month, beforeyesterday.day))
             query = sa.select([func.count(reqs_tbl.c.id).label("totalbuilds")])\
-                .where(reqs_tbl.c.submitted_at >= lastday)\
+                .where(reqs_tbl.c.complete_at >= beforeyesterday_epoch)\
+                .where(reqs_tbl.c.complete_at <= yesterday_epoch)\
```

Comment:
So why do we need to know the BRs between the day before yesterday and yesterday?